### ISSUE TITLE
Integration with rq-scheduler?

### ISSUE THREAD

**Issue by lost-theory** [Parallels/rq-dashboard#70]
Hello,

Do you know if there is any work on integrating rq-dashboard with rq-scheduler? Would you accept a PR?

For instance, if a conditional import for `rq_scheduler` works, then a new section would be added to the web interface containing details on scheduled jobs. It would be optional.

It may also be possible to take the blueprint from this package and use it in a new flask app that contains the rq-scheduler functionality (as a separate package or something, if you don't want rq-scheduler code to live in this project).

Anyway, just wanted to see what other folks thought, since it'd be nice to see both in one web interface.

**Comment by mtsgrd** [comment]
What's the best way of making additional charts and tools pluggable?

**Comment by tyler274** [comment]
This would be fantastic.

**Comment by joostdevries** [comment]
I agree 100% that this would be awesome. To avoid having a dependency on `rq-scheduler`, it might be best to actually pass an instance of `Scheduler` to the dashboard.

**Comment by tcorbettclark** [comment]
I think an "rq-scheduler-dashboard" should be a separate Flask blueprint from the current rq-dashboard, and should probably be a separate package at least initially.

If rq-scheduler became closer to rq then this would motivate bringing the dashboards together too. But the fact that it was easy and clean to add the scheduler on top of rq says something. Basically I think each dashboard should be free to track the features of the corresponding underlying component.

So if someone first made a standard alone scheduler dashboard in a separate Flask blueprint, we could then see how well the two sat next to one another before thinking about bringing them closer. It needs a first date!!

**Comment by lamflam** [comment]
I had a need for this as well so I took the changes from @helenst made in #95 and wrapped them up into a separate [`rq-scheduler-dashboard`](https://github.com/lamflam/rq-scheduler-dashboard) package. All credit to @helenst, this dashboard is very useful!

**Comment by ducu** [comment]
Can you provide a screenshot for this dashboard?

**Comment by lamflam** [comment]
@docu Added a screenshot to the readme.

**Comment by ducu** [comment]
Looks good, I also made some tweaks to the dashboard that I found useful (https://github.com/ducu/rq-dashboard/pull/94), it would be nice to harmonize and merge these improvements. Vincent transferred this repo to me, I would be happy to get things moving, pull requests are welcome.
